test/steps/1_given.js

```diff
@@ -39,6 +39,14 @@ import {
 
 const envToStub = require('../../src/utils/env');
 
+export function anAlias() {
+	this.test.ctx.alias = getFirstQuotedString(this.test.parent.title);
+}
+
+export function aLiskTransactionObject() {
+	this.test.ctx.createLiskTransaction = createLiskTransaction;
```

Comment:
The name diverges from the title, which suggests one should change (see comment above).